meilisearch/meilisearch_api_error.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+class MeiliSearchApiError(Exception):
+    """Error from MeiliSearch API"""
+    def __init__(self, error, request):
+        self.status_code = request.status_code
+        if request.text:
+            self.message = f'{json.loads(request.text)["message"]}'
```

Comment:
What happens if the is no "message" in the body? (it should not happen but it would be bad if there is an exception in the exception ^^ There was a time where MeiliSearch did not return any message for some HTTP error = 4XX or 5XX)

Reply:
Right, should check if there's a message before setting it to the internat attribute. And have a default value like "No message received" for example

Reply:
Don't you think that the user would believe that there is an error because "there is no message" with this default value? @eskombro

Reply:
Maybe a simple `unknown` can solve that?